run-pa11y.sh

```diff
@@ -15,19 +14,44 @@ lduId=$2
 teamId=$3
 omId=$4
 
+pa11yCommand="""pa11y --standard WCAG2AA --ignore "warning\;notice" --hide-elements "div[role=presentation]" """
+host="http://localhost:3000"
```

Comment:
Can we pull the value of the base path from configuration so it can be run independently on each environment?